### DISCUSSION TITLE
How to create a product with a combination in Prestashop 8 through API

### DISCUSSION THREAD

**Discussion by hardikiflair** [PrestaShop/PrestaShop#34637] [Q&A]
I am using Prestashop API for product creation but in Prestashop version 8 product create without combination. 

kindly check and please guide how to fix combination api issue

Looking forward to your response.

Thank you.

**Comment by kpodemski** [comment]
Hello @hardikiflair 

There's a full documentation here:
https://devdocs.prestashop-project.org/8/webservice/tutorials/create-product-az/

There are also example PostMan collections here in this repository:
https://github.com/PrestaShop/webservice-postman-examples/tree/main

I hope that helps :)
